lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -59,13 +59,15 @@ export default function StudentSelector({
 
     return (
       <span className="StudentsSelector__students">
-        {/* FIXME-A11Y */}
-        {/* eslint-disable-next-line jsx-a11y/no-onchange */}
         <select
           className="StudentsSelector__students-select"
           onChange={e => {
             onSelectStudent(parseInt(e.target.value));
           }}
+          // See https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md
+          onBlur={e => {
```

Comment:
I found this has no ill wanted side effect, but I still don't like it, I'm almost leaning towards adding the lint disable comment back. Thoughts?

Reply:
According to a [comment in the MDN docs](https://hyp.is/XcWxEFPMEeq0tYNWatw4eA/developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) the "change" event is now triggered consistently across all modern browsers (including Firefox as of v63, from October 2018). On that basis, assuming `onChange` is triggered when we want it to be, when using either keyboard or mouse navigation, I'd be OK with putting the lint disable comment back. Do include a note about why we're ignoring it if so.